src/nasui/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Is this file useful?

Reply:
Yes.

Reply:
@ultmaster better to explain why

Reply:
It's explained in https://github.com/facebook/create-react-app/issues/6560. I guess we should respect the practice of create-react-app.